google/cloud/firestore_v1/query.py

```diff
@@ -506,3 +566,122 @@ def get_partitions(
             start_at = cursor
 
         yield QueryPartition(self, start_at, None)
+
+
+class Iterator(page_iterator.Iterator):
```

Comment:
also: I know this is early, but I'd probably prefer to give the iterator a more descriptive name

Reply:
I tentatively changed the name to `StreamIterator`. Maybe we can find a better name, such as `DocsIterator`?